### ISSUE TITLE
Feature request: SpeedDial

### ISSUE THREAD

**Issue by Exley2149** [fork-maintainers/iceraven-browser#666]
even though in addons it is shown that you can "install" some sort of speeddial with iceraven, but you can't...
i think this is a must feature for any browser... i'm still surprised FF doesn't have it by default, but screw them then
make Iceraven have it, pretty please.

**Comment by Exley2149** [comment]
no it is not same
dials can be arranged freely on any space and even within subfolder, pins cannot be moved at all and they only get pinned in strict order... they are useless

**Comment by MilaMurry** [comment]
Agree. Those shortcuts need rework, like multiple rows/columns, editing URLs, and rearrangement.
